src/TicketingService.Abstractions/TicketResult.cs

```diff
@@ -13,11 +14,16 @@ public record TicketResult
     [JsonInclude]
     [JsonPropertyName("json")]
     [DataMember(Name = "json")]
-    public string? ResultAsJson { get; set; }
+    public string? ResultAsJson { get; set; } = null;
 
     public TicketResult(object? result)
     {
-        ResultAsJson = result is not null ? JsonSerializer.Serialize(result) : null;
+        if (result is not null)
+        {
+            ResultAsJson = result is not Array
```

Comment:
Why do you wrap the object in an array? Isn't it enough that you do `Task Error(Guid ticketId, TicketError error, CancellationToken cancellationToken = default) => Error(ticketId, new [] { error }, cancellationToken);` ? By also wrapping it here, you also influence non-error ticket results.